src/components/my-widgets-copy-dialog.jsx

```diff
@@ -1,27 +1,30 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 import Modal from './modal'
+import './my-widgets-copy-dialog.scss'
 
-const MyWidgetsCopyDialog = ({onClose, onCopy}) => {
-	const [newTitle, setNewTitle] = useState('Name of Widget (Copy)')
+const MyWidgetsCopyDialog = ({onClose, onCopy, name}) => {
+	const [newTitle, setNewTitle] = useState('')
 	const [copyPermissions, setCopyPermissions] = useState(false)
+
+	// Sets the initial title
+	useEffect(() => {setNewTitle(name + " copy")}, [])
```

Comment:
Can I suggest using the parenthesis?

```suggestion
	useEffect(() => {setNewTitle(name + " (Copy)")}, [])
```